src/test/TestGhoStewards.t.sol

```diff
@@ -40,45 +65,231 @@ contract TestGhoStewards is Test {
+    vm.prank(RISK_COUNCIL);
+    GHO_AAVE_STEWARD.updateGhoBorrowRate(
+      currentRates.optimalUsageRatio,
+      currentRates.baseVariableBorrowRate + 1,
+      currentRates.variableRateSlope1,
+      currentRates.variableRateSlope2
+    );
+    assertEq(_getGhoBorrowRate(), currentRates.baseVariableBorrowRate + 1);
+  }
+
+  function testGhoBucketStewardUpdateFacilitatorBucketCapacity() public {
+    (uint256 currentBucketCapacity, ) = GhoToken(GHO_TOKEN).getFacilitatorBucket(
+      address(GHO_ATOKEN)
+    );
+    vm.prank(RISK_COUNCIL);
+    uint128 newBucketCapacity = uint128(currentBucketCapacity) + 1;
+    GHO_BUCKET_STEWARD.updateFacilitatorBucketCapacity(address(GHO_ATOKEN), newBucketCapacity);
+    (uint256 capacity, ) = GhoToken(GHO_TOKEN).getFacilitatorBucket(address(GHO_ATOKEN));
+    assertEq(newBucketCapacity, capacity);
+  }
+
+  function testGhoBucketStewardSetControlledFacilitatorAdd() public {
+    address[] memory oldControlledFacilitators = GHO_BUCKET_STEWARD.getControlledFacilitators();
+    address[] memory newGsmList = new address[](1);
+    address gho_gsm_4626 = makeAddr('gho_gsm_4626');
+    newGsmList[0] = gho_gsm_4626;
+    vm.prank(OWNER);
+    GHO_BUCKET_STEWARD.setControlledFacilitator(newGsmList, true);
+    address[] memory newControlledFacilitators = GHO_BUCKET_STEWARD.getControlledFacilitators();
+    assertEq(newControlledFacilitators.length, oldControlledFacilitators.length + 1);
+    assertTrue(_contains(newControlledFacilitators, gho_gsm_4626));
+  }
+
+  function testGhoCcipStewardUpdateBridgeLimit() public {
+    uint256 oldBridgeLimit = IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).getBridgeLimit();
+    uint256 newBridgeLimit = oldBridgeLimit + 1;
+    vm.prank(RISK_COUNCIL);
+    GHO_CCIP_STEWARD.updateBridgeLimit(newBridgeLimit);
+    uint256 currentBridgeLimit = IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).getBridgeLimit();
+    assertEq(currentBridgeLimit, newBridgeLimit);
+  }
+
+  function testGhoCcipStewardUpdateRateLimit() public {
+    RateLimiter.TokenBucket memory outboundConfig = MockUpgradeableLockReleaseTokenPool(
+      GHO_TOKEN_POOL
+    ).getCurrentOutboundRateLimiterState(remoteChainSelector);
+    RateLimiter.TokenBucket memory inboundConfig = MockUpgradeableLockReleaseTokenPool(
+      GHO_TOKEN_POOL
+    ).getCurrentInboundRateLimiterState(remoteChainSelector);
+
+    RateLimiter.Config memory newOutboundConfig = RateLimiter.Config({
+      isEnabled: outboundConfig.isEnabled,
+      capacity: outboundConfig.capacity + 1,
+      rate: outboundConfig.rate
+    });
+
+    RateLimiter.Config memory newInboundConfig = RateLimiter.Config({
+      isEnabled: outboundConfig.isEnabled,
+      capacity: inboundConfig.capacity,
+      rate: inboundConfig.rate
+    });
+
+    // Currently rate limit set to 0, so can't even change by 1 because 100% of 0 is 0
+    vm.expectRevert('INVALID_RATE_LIMIT_UPDATE');
```

Comment:
revisit spec

Reply:
This is correct, going from zero to non-zero is something that requires AIP process.